core/rfb.js

```diff
@@ -137,6 +138,7 @@ export default class RFB extends EventTargetMixin {
         this._keyboard = null;          // Keyboard input handler object
         this._gestures = null;          // Gesture input handler object
         this._resizeObserver = null;    // Resize observer object
+        this._rfbRA2State = null;
```

Comment:
Authentication related attributes are further up. Better to keep it there to clarify what it is used for.